modules/installation-about-restricted-network.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_restricted_networks/installing-restricted-networks-preparations.adoc
+
+[id="installation-about-restricted-networks_{context}"]
+= About installations in restricted networks
+
+In {product-title} {version}, you can perform an installation that does not
+require an active connection to the internet to obtain software components. You
+complete an installation in a restricted network on only infrastructure that you provision,
+not infrastructure that the installer provisions, so your platform selection is
+limited.
+// maybe point out that you can follow the bare metal installation rules on supported hardware and link to the matrix
+
+If you choose to perform a restricted network installation on a cloud platform, you
+still require access to its cloud APIs. Some cloud functions, like
+Amazon Web Service's IAM service, require internet access, so might still
+require internet access.
+//behind a proxy
+Depending on your network, you might require less internet
+access for an installation on bare metal hardware or on VMware vSphere.
+
+To complete a restricted network installation, you must create a repository that
+mirrors the contents of the {product-title} registry and contains the
+installation media. You can create this mirror on a bastion host, which can
+access both the internet and your closed network, or by using other methods
+that meet your restrictions.
+
+[IMPORTANT]
+====
+Restricted network installations always use user-provisioned infrastructure.
+Because of the complexity of the configuration for user-provisioned installations,
+consider completing a standard user-provisioned infrastructure installation before
+you attempt a restricted network installation. Completing this test installation might
+make it easier to isolate and troubleshoot any issues that might arise
+during your installation in a restricted network.
+====
+
+[id="installation-restricted-network-limits{context}"]
+== Additional limits
+
+Clusters in restricted networks have the following additional limitations and restrictions:
+
+* The ClusterVersion status includes an `Unable to retrieve available updates`
+error.
+//* The authentication Operator might randomly fail.
+* By default, you cannot use the contents of the Developer Catalog because
+ you cannot access the required ImageStreamTags.
+//* The `TelemeterClientDown` and `Watchdog` alerts from the monitoring Operator always display.
+
+[id="installation-restricted-network-mirror_{context}"]
+== About the mirror repository
+
+The key component that is require to complete an installation in a restricted
```

Comment:
nit: "require" -> "required".  Or maybe "The key component that is required to complete an installation" -> "The key to installation" or some such.

Reply:
I'd take "The key to installation, but because we mean "key" as an adjective here, we have to make the nouns explicit. It's a requirement in our style guide that helps with translation and improves clarity for non-native English speakers.